apm-sniffer/apm-sdk-plugin/dubbo-2.7.x-plugin/pom.xml

```diff
@@ -32,7 +32,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dubbo.version>2.7.0</dubbo.version>
+        <dubbo.version>2.7.15</dubbo.version>
```

Comment:
I don't think this is the correct change. Java agent plugin covers different versions. You change the codes according to the latest version, which makes everyone using < 2.7.14 + SkyWalking agent 8.9.0+ faces unexpected bugs.

Reply:
This kind of API behavior change is strange in this tiny bug-fix version change. Could you report to Dubbo community for an explanation?

Reply:
If this API exists for all previous 2.7.x versions. please don't upgrade this dependency version.

Reply:
Agree